bundles/binding/org.openhab.binding.isy/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/main/java/,\
+           src/main/resources/
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               lib/isy_inst.jar
```

Comment:
Hi Mitch,

Yes of course. The only caveat is to keep up with official releases. You have a free license to include it in as many projects as you wish as long as its purpose is not defamation, terrorism, and/or otherwise hurtful to others.

With kind regards,
**********************************************


 With kind regards,

*****************************************
UDI Tech Support  
Michel Kohanim
Universal Devices, Inc.
support@universal-devices.com

Information (Wiki):     http://wiki.universal-devices.com
Information (Forum): http://forum.universal-devices.com
Submit a Ticket:         http://www.universal-devices.com/contact-support